### ISSUE TITLE
Theme-free style

### ISSUE THREAD

**Issue by whistyun** [whistyun/Markdown.Avalonia#19]
Embedded styles foreground and background are set hardcode color.
If I use dark theme, some text is not visible.

**Comment by whistyun** [comment]
I made new style 'Standard2'

![image](https://user-images.githubusercontent.com/13712028/102679708-04e76a80-41f5-11eb-9143-35d86cca97a1.png)

![image](https://user-images.githubusercontent.com/13712028/102679722-1d578500-41f5-11eb-957f-7dded93f826a.png)

**Comment by whistyun** [comment]
this feature released in 0.10.0-a8 and 0.9.0-a8.

The style chosen by this feature is subtly different from the previous ones.

If you like the old ones, change the Markdown Scroll Viewer settings.

```xml
<md:MarkdownScrollViewer
                 MarkdownStyleName="Standard" 
                 />
```
